src/main/kotlin/step2/LottoApplication.kt

```diff
@@ -0,0 +1,23 @@
+package step2
+
+import step2.domain.*
+import step2.ui.InputView
+import step2.ui.OutputView
+
+fun main() {
+    val purchaseAmount = InputView.getPurchaseAmount()
+    val lottoPurchaseCount = InputView.getPurchaseCount(purchaseAmount)
+
+    val lottoList = (1..lottoPurchaseCount).map {
+        Lotto(RandomGenerateStrategy().of())
+    }
```

Comment:
```suggestion
    val lottoList = List(lottoPurchaseCount) {
        Lotto(RandomGenerateStrategy().of())
    }
```
다음 인라인 함수를 활용하여 리스트를 생성하면 불필요한 range 인스턴스를 만들지 않아도 됩니다.